cloud/kubernetes/cockroachdb-petset.yaml

```diff
@@ -59,6 +66,33 @@ spec:
         app: cockroachdb
       annotations:
         pod.alpha.kubernetes.io/initialized: "true"
+        pod.alpha.kubernetes.io/init-containers: '[
```

Comment:
Is there a link to some docs you could put in here to explain "how things work"? In particular, `cockroach-k8s-init` will be run with what parameters and what stdin?

The `cockroach-k8s-init` container has an entry point `/peer-finder`, but then there's also the `on-start.sh` script which does the actual work but it's not clear how it's invoked. A short overview would be helpful.